projects/core/abstract/abstract-textfield-host.ts

```diff
@@ -32,7 +32,7 @@ export abstract class AbstractTuiTextfieldHost<T extends AbstractTuiControl<any>
         return this.host.value?.toString() || ``;
     }
 
-    abstract onValueChange(value: string): void;
+    abstract onValueChange(value: string[] | string): void;
```

Comment:
I'm a bit skeptical about changing the very base class. Are you sure it's necessary and it doesn't bring any confusion to other usecases?

Reply:
Hmm, ok I reverted changes. Plz pay attention - now we need to implement empty `onValueChange` and separate `onSelectionChange(string[])` in the `TuiMultiSelectDirective`. This way we don't touch base class, ok?

Reply:
Ok, let's keep it that way